xds/internal/clusterspecifier/rls/rls.go

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package rls implements the RLS cluster specifier plugin.
+package rls
```

Comment:
Should this be importing the RLS package?  Seems like it should.

Reply:
I did, but it failed vet. Let me try again.

Reply:
But I agree, that way the import dependency graph through the xdsclient gets this balancer eventually registered.